sklearn/cluster/dbscan_.py

```diff
@@ -184,6 +190,11 @@ class DBSCAN(BaseEstimator, ClusterMixin):
         .. versionadded:: 0.17
            metric *precomputed* to accept precomputed sparse matrix.
 
+   metric_params : dict, optional (default = None)
```

Comment:
I think "optional" is sufficient when the semantics of the default are not clear.